packages/odyssey-storybook/package.json

```diff
@@ -13,12 +13,12 @@
   "devDependencies": {
     "@applitools/eyes-storybook": "^3.29.4",
     "@babel/core": "^7.19.3",
-    "@okta/odyssey-babel-loader": "^0.17.1",
-    "@okta/odyssey-design-tokens": "^0.17.1",
-    "@okta/odyssey-lifecycle": "^0.17.1",
-    "@okta/odyssey-react-labs": "^0.17.1",
-    "@okta/odyssey-react-mui": "^0.17.1",
-    "@okta/odyssey-typescript": "^0.17.1",
+    "@okta/odyssey-babel-loader": "*",
```

Comment:
`workspace:*`

Reply:
Typescript complains when I switch to `workspace:*`
```
Executing command in 3 packages: "yarn run typecheck"
TypeError: Cannot read properties of undefined (reading 'version')
```

Reply:
... would need to dig in a lot more into how this repo is setup to understand why. Perhaps going back to origin notation? Not sure what the intent of using '*' is, and why in a yarn 3 repo 'workspace:*' does not work.

Reply:
Sure. Using `*` so that the new `TextField` story don't need to wait until new `TextField` is published.

Reply:
@macyabbey-okta updated. using yarn workspace command to replace lerna cz lerna not recognizing `workspace:*`.